libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/comboChart/PluggableComboChart.tsx

```diff
@@ -235,6 +244,76 @@ export class PluggableComboChart extends PluggableBaseChart {
         );
     }
 
+    protected getDefaultAndAvailableSort(
+        buckets: IBucketOfFun[],
+        properties: IVisualizationProperties,
+    ): {
+        defaultSort: ISortConfig["currentSort"];
+        availableSorts: ISortConfig["availableSorts"];
+    } {
+        const measures = getBucketItemsByType(buckets, BucketNames.MEASURES, [METRIC]);
+        const secondaryMeasures = getBucketItemsByType(buckets, BucketNames.SECONDARY_MEASURES, [METRIC]);
+        const viewBy = getBucketItems(buckets, BucketNames.VIEW);
+        const stackMeasures = properties?.controls?.stackMeasures;
```

Comment:
I had an issue with this line in area chart. AD enabled it based on UI config but did not send `true` via this property unless I clicked upon it explictly. It seems that combo chart should behave the same but something is broken in AD and it is not enabled by default. Based on this method called from `getUiConfig` it should be by default enabled. https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/comboChart/PluggableComboChart.tsx#L126

I have there a fallback setup as follows: `const canSortStackTotal = properties?.controls?.stackMeasures ?? this.getUiConfig().optionalStacking.stackMeasures;` I'm not sure if it is necessary here or of there is some side effect.